src/Resources/views/Admin/UiElement/youtube.html.twig

```diff
@@ -5,15 +5,19 @@
         link
         width
         height
+        align
 #}
 {# Don't display video if URL is not correct (empty) #}
 {% set url = monsieurbiz_richeditor_youtube_link(element.link) %}
+{% set align = element.align is defined and element.align is not empty ? element.align : 'inherit' %}
 {% if url is not null %}
-    <iframe
-        width="{{ element.width ?: 560 }}"
-        height="{{ element.height ?: 315 }}"
-        src="{{ monsieurbiz_richeditor_youtube_link(element.link) }}"
-        frameborder="0"
-        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
-    </iframe>
+    <div style="text-align: {{align}};">
```

Comment:
Is the text-align going to do something on an iframe?

Reply:
text-align on iframe does nothing I have to encapsulate it with a div